scripts/app_sre_pr_check.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
 set -exv
 
-BASE_IMG="quay"
-
-IMG="${BASE_IMG}:latest"
-
-BUILD_CMD="docker build" IMG="$IMG" make app-sre-docker-build
+# DO NOTHING
```

Comment:
File needed?

Reply:
The file is needed for now because the app-sre pipeline looks for it. Once the PR is merged, I will be able to update the Jenkins job and remove this file.